cmake/Modules/FindMKLDNN.cmake

```diff
@@ -85,8 +87,12 @@ ELSE(MKL_FOUND)
       MESSAGE(STATUS "MKL-DNN not found. Compiling without MKL-DNN support")
       RETURN()
     ENDIF(NOT ${__mklml_inner_lib_upper}_LIBRARY)
-    LIST(APPEND MKLDNN_LIBRARIES ${${__mklml_inner_lib_upper}_LIBRARY})
+    FILE(COPY ${${__mklml_inner_lib_upper}_LIBRARY} DESTINATION ${CMAKE_BINARY_DIR}/lib)
+    INSTALL(FILES ${${__mklml_inner_lib_upper}_LIBRARY} DESTINATION lib
+      PERMISSIONS OWNER_EXECUTE OWNER_WRITE OWNER_READ GROUP_EXECUTE GROUP_READ WORLD_EXECUTE)
   ENDFOREACH(__mklml_inner_lib)
+  SET(MKLROOT ${CMAKE_BINARY_DIR})
```

Comment:
Is `MKLROOT` used?

Reply:
I see it's used down next level. nvm.

Reply:
Yes. Will be used by MKL-DNN to find MKLML path.
It's OK to set MKLROOT for MKLML here. It will be set only if pre-installed MKL is not found.